package.json

```diff
@@ -32,19 +32,19 @@
         "node": ">=6"
     },
     "dependencies": {
-        "@types/glob": "5 - 7",
-        "@types/parse5": "^5",
         "css-selector-parser": "^1.3",
         "glob": "5 - 7",
         "parse5": "^5",
         "pofile": "^1",
```

Comment:
There's a trailing comma here, which is why the Travis build failed

Reply:
Ack sorry, fixing.